index.ts

```diff
@@ -59,20 +53,6 @@ app.get('/rne/:siren', rneControllerAPI);
  */
 app.get('/rne/fallback/:siren', rneControllerSite);
 
-// list documents
-app.get('/rne/documents/:siren', rneListDocumentsController);
-
-// download an acte
-app.get('/rne/download/acte/:id', rneActeDownloadController);
-
-// download a bilan
-app.get('/rne/download/bilan/:id', rneBilanDownloadController);
-
-/**
- * Status
- */
-app.use('/status', statusRouter);
```

Comment:
I would keep /status and use the monitor for the open data client

Reply:
I added a vat sonde to keep the code